Sample Applications/CustomComboBox/MainWindow.xaml

```diff
@@ -21,7 +21,7 @@
                              ItemsSource="{Binding Movies}" Command="{Binding OnWatchNow}" 
                              ItemContainerStyle="{StaticResource MovieItemContainerStyle}"
                              DisplayMemberPath="Title"
-                             IsTabStop="False"/>
+                             IsTabStop="False" Width="160"/>
```

Comment:
This is needed because when changing the visibility to colapsed, it reduces the size, right? Have you tried using Hidden instead of Collapsed? I think that would not ve needed with it.